packages/quantic/force-app/main/default/lwc/quanticUtils/quanticUtils.js

```diff
@@ -88,4 +137,14 @@ export const keys = {
   ENTER: 'Enter',
   ARROWUP: 'ArrowUp',
   ARROWDOWN: 'ArrowDown',
-};
\ No newline at end of file
+};
+
+export function getItemfromLocalStorage(key) {
+  const value = localStorage.getItem(key);
+  return value !== 'undefined' ? JSON.parse(value) : null;
+}
+
+export function setIteminLocalStorage(key, item) {
+  const value = JSON.stringify(item);
+  localStorage.setItem(key, value);
+}
```

Comment:
Given the key is static and not bound to an engine, donc we risk polution from different search interfaces on the same domain ?

Reply:
That's a very good point. I'll add the engine-id in the key!